### ISSUE TITLE
Hi author,Please Read me and help me

### ISSUE THREAD

**Issue by xiaonian233** [erycson/BloodStrike#1]
Author ,  I love your Bloodstrike Emulador , But i can't build BloodStrikeServer , I have already solved all problems ,  ![image](https://cloud.githubusercontent.com/assets/21072615/17723336/f27ffb34-646b-11e6-83b1-e038c0aba199.png),I export jar can't work , Please help me to build it?
Thank you very much

**Comment by erycson** [comment]
You are giving an error? I have not tried exporting yet, I'll try and see if I can. If I can say what were the steps to follow

**Comment by xiaonian233** [comment]
OK,thank you!

2016-08-17 12:27 GMT+08:00 Érycson Gomes da Nóbrega <
notifications@github.com>:

**Comment by erycson** [comment]
Try to delete the test files and then follow this tutorial: https://www.youtube.com/watch?v=rpZiSi-bP5Q

**Comment by xiaonian233** [comment]
Thank you author

**Comment by xiaonian233** [comment]
I have already build it , build a .jar file , However,how i run it ? when i run it with "java -jar lobbyplayer-0.0.1-SNAPSHOT.jar' it tells me ![image](https://cloud.githubusercontent.com/assets/21072615/17761556/9e0db7d8-653a-11e6-8328-bf6446a82c7a.png) , but i've import log4j-1.2.17

**Comment by erycson** [comment]
see the server configuration file or the log4j configuration file, one of the two is missing

**Comment by xiaonian233** [comment]
.classpath file:![image](https://cloud.githubusercontent.com/assets/21072615/17805353/c3ea72be-6631-11e6-8996-79494d0bbb48.png)
pom.xml:
![image](https://cloud.githubusercontent.com/assets/21072615/17805364/d568068c-6631-11e6-8d1a-3055a95ca9fc.png)

**Comment by xiaonian233** [comment]
I don't think it is missing
![image](https://cloud.githubusercontent.com/assets/21072615/17805675/32503fc6-6633-11e6-991a-664dd4ddd029.png)

**Comment by xiaonian233** [comment]
Author,could you send your build .jar file to me ? I've solved that problem then new problem , when i use client join, it tells me 

![image](https://cloud.githubusercontent.com/assets/21072615/17807426/14179edc-663d-11e6-83d2-fc25c7c3c245.png)

**Comment by erycson** [comment]
I sent the commit 2eebdd57b1ab30d95429b1dc9f2f798550d5edd8 to correct this build. Run `build.bat` to compile.

**Comment by xiaonian233** [comment]
OK，Must i use Mysql?

**Comment by xiaonian233** [comment]
When i click  ![image](https://cloud.githubusercontent.com/assets/21072615/17828795/4b2d330a-66ce-11e6-8276-0d700c0586c3.png),server return  ![image](https://cloud.githubusercontent.com/assets/21072615/17828800/5ff99d82-66ce-11e6-994f-b60942740f08.png)

**Comment by erycson** [comment]
Yes, it uses MySQL, but the server is not complete. The problem is sending login to the client.

**Comment by xiaonian233** [comment]
Do you have a way to solve it

**Comment by erycson** [comment]
Sorry, but until now never figure out how to solve this

**Comment by erycson** [comment]
But his mistake, I believe is the lack of tables. JPA change settings for it to create the tables automatically, I think this will solve your problem.

**Comment by xiaonian233** [comment]
OK,i wll try it , thank you again

**Comment by LapisBlock** [comment]
Can you gave me your jar?
